mesa_geo/visualization/modules/MapVisualization.py

```diff
@@ -45,7 +45,7 @@ class MapModule(VisualizationElement):
         "css/external/leaflet.css",
         "js/external/leaflet.js",
     ]
-    local_dir = os.path.dirname(__file__) + "/../templates"
+    local_dir = (Path(__file__).parent / "/../templates").resolve()
```

Comment:
Should it be `"../templates"` instead of `"/../templates"` (removing the leading `/`)?

Reply:
It's technically equivalent, but confusing to leave it in there, so yes, we should take that out.  Thank you!